src/libraries/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -28,6 +28,9 @@ internal static partial class JsonConstants
         public const byte UtcOffsetToken = (byte)'Z';
         public const byte TimePrefix = (byte)'T';
 
+        public const string NewLineLineFeed = "\n";
+        public const string NewLineCarriageReturnLineFeed = "\r\n";
```

Comment:
Nit: kind of redundant naming?
```suggestion
        public const string LineFeed = "\n";
        public const string CarriageReturnLineFeed = "\r\n";
```

Reply:
LineFeed is already taken on line 14 as a byte constant.

Reply:
Do we even need these to be named?

Reply:
I was just following the existing pattern based off of the `TabIndentCharacter` and `DefaultIndentCharacter` constants and their usage.

I can just inline them if preferred, both of these new constants are used in two places each.